src/components/Button.js

```diff
@@ -56,21 +64,25 @@ const Button = (props) => {
             return <ContentComponent />;
         }
 
-        return props.isLoading
-            ? (
-                <ActivityIndicator color={themeColors.textReversed} />
-            ) : (
-                <Text
-                    selectable={false}
-                    style={[
-                        styles.buttonText,
-                        props.success && styles.buttonSuccessText,
-                        ...props.textStyles,
-                    ]}
-                >
-                    {props.text}
-                </Text>
-            );
+        return (
+            <>
```

Comment:
For my own understanding, was this change just to correct the style/formatting?

Reply:
No, we don't really need this change. I had it wrapped in a `View` at one point and forgot to clean this up.